<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportHisData</class>
 <widget class="QWidget" name="ExportHisData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lbTitle">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>历史数据导出</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbStartYear">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>130</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>年</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbStartYear_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>130</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>月</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbStartYear_3">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>130</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>日</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbStartYear_4">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>180</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>秒</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbStartYear_5">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>180</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>分</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbStartYear_6">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>180</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>时</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbIniDate">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>起始日期：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leStartYear">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leStartMonth">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>130</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leStartDay">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>130</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leStartHour">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>180</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leStartMinute">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>180</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leStartSecond">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>180</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbStartYear_7">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>320</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>秒</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbOverDate">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>结束日期：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbStartYear_8">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>320</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>时</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbStartYear_9">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>270</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>月</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leOverYear">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>270</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leOverDay">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>270</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leOverMonth">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>270</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leOverHour">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>320</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leOverSecond">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>320</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbStartYear_10">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>270</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>日</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbStartYear_11">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>320</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>分</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leOverMinute">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>320</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbStartYear_12">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>270</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>年</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>591</width>
     <height>121</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>2</width>
      <height>121</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>591</width>
     <height>121</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>2</width>
      <height>121</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pbExport">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>400</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>导出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbExit">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>400</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>frame_2</zorder>
  <zorder>lbTitle</zorder>
  <zorder>lbStartYear</zorder>
  <zorder>lbStartYear_2</zorder>
  <zorder>lbStartYear_3</zorder>
  <zorder>lbStartYear_4</zorder>
  <zorder>lbStartYear_5</zorder>
  <zorder>lbStartYear_6</zorder>
  <zorder>lbIniDate</zorder>
  <zorder>leStartYear</zorder>
  <zorder>leStartMonth</zorder>
  <zorder>leStartDay</zorder>
  <zorder>leStartHour</zorder>
  <zorder>leStartMinute</zorder>
  <zorder>leStartSecond</zorder>
  <zorder>lbStartYear_7</zorder>
  <zorder>lbOverDate</zorder>
  <zorder>lbStartYear_8</zorder>
  <zorder>lbStartYear_9</zorder>
  <zorder>leOverYear</zorder>
  <zorder>leOverDay</zorder>
  <zorder>leOverMonth</zorder>
  <zorder>leOverHour</zorder>
  <zorder>leOverSecond</zorder>
  <zorder>lbStartYear_10</zorder>
  <zorder>lbStartYear_11</zorder>
  <zorder>leOverMinute</zorder>
  <zorder>lbStartYear_12</zorder>
  <zorder>pbExport</zorder>
  <zorder>pbExit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
